apps/codeforafrica/src/theme/index.js

```diff
@@ -0,0 +1,282 @@
+import { createTheme } from "@commons-ui/core";
+
+const FONT_FAMILY_PRIMARY = "'Open Sans', sans-serif";
+const FONT_FAMILY_SECONDARY = "'Merriweather', serif";
+
+const palette = {
+  mode: "light",
+  primary: { main: "#1020E1", light: "#EFF0FD", dark: "#08125C" },
+  secondary: { main: "#000000", light: "#7F7272" },
+  text: {
+    primary: { main: "#000000" },
+    secondary: { main: "#FFFFFF" },
+  },
+  background: { main: "#F6F5F5" },
+  highlight: { main: "#ED1C24" },
+  grey: { main: "#B4ABAB", light: "#E3DFDF" },
+};
+
+const breakpoints = {
+  values: {
+    xs: 390,
+    sm: 768,
+    md: 1152,
+    lg: 1440,
+    xl: 1920,
+  },
+};
+
+const theme = createTheme({
+  palette,
+  breakpoints,
+  typography: {
+    FONT_FAMILY_PRIMARY,
+    FONT_FAMILY_SECONDARY,
```

Comment:
```suggestion
    fontFamily: FONT_FAMILY_PRIMARY,
```
Or I'm reading it wrong?

Reply:
@kilemensi Um, naaah. I think it should be like that as well.